support/cas-server-support-pac4j-core-clients/build.gradle

```diff
@@ -3,7 +3,11 @@ dependencies {
     implementation project(":core:cas-server-core-tickets-api")
     implementation project(":core:cas-server-core-web-api")
     implementation project(":core:cas-server-core-authentication-api")
+    implementation project(":core:cas-server-core-util")
+    implementation project(":core:cas-server-core-configuration")
     implementation project(":support:cas-server-support-pac4j-core")
+    implementation project(":support:cas-server-support-actions")
```

Comment:
Why import this?

Reply:
Right, this was indeed useless, dropped. Thanks!